test/pageobjects/HomePage.js

```diff
@@ -2,38 +2,72 @@ import Base from './Base'
 
 class HomePage extends Base {
     get ToolsQaImg() { return $('img') }
-    get h5() { return $('h5')}
-    get ElementsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(1)') }
-    get FormsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(2)') }
-    get AFWEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(3)') } 
-    get WidgetsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(4)') }
-    get InteractionsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(5)') }
-    get BookStoreAppEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(6)') }
```

Comment:
Avoid using xPath selectors, this kid of selectors are complicated to maintain in time. Use type, css, atribute, ID selectors instead.